projects/ngx-meta/src/core/src/metadata-values.ts

```diff
@@ -1,3 +1 @@
-export interface MetadataValues {
-  [key: string]: unknown
-}
```

Comment:
This was giving issues, as it's a specific type of `object`. An object that implements an interface without index signature wouldn't comply with this. Despite yet being valid metadata values. So going for a broader definition